core/src/main/java/io/blockv/core/client/manager/VatomManager.kt

```diff
@@ -33,13 +33,18 @@ interface VatomManager {
   fun getVatoms(vararg ids: String): Callable<Group>
 
   /**
-   * Fetches the current users inventory.
+   * Fetches the current user's inventory of vAtoms.
    *
-   * @param id is the id of the inventory you want to fetch.
+   * @param id is the id of the inventory you want to fetch. If null or '.' supplied the
+   *           user's root inventory will be returned.
+   * @param page indicates which slice of the vAtom inventory is returned. If set as
+   *             zero, the first page is returned.
+   * @param limit defines the number of vAtoms per response page (up to 100). If omitted or set as
+   *              zero, the max number is returned.
    * @return new Callable<Group> instance.
    * @see Group
    */
-  fun getInventory(id: String?): Callable<Group>
+  fun getInventory(id: String?, page: Int, limit: Int): Callable<Group>
```

Comment:
You have called the parameter `id`, on iOS, it is called `parentID`. Should we align?